src/argvalues.rs

```diff
@@ -77,6 +95,27 @@ impl ArgValues {
         let wrap2 = matches.value_of("wrap2").map(&str_to_wrapmode);
         let wrap3 = matches.value_of("wrap3").map(&str_to_wrapmode);
 
+        // Anistropic filter max value
+        let anisotropic_max = matches.value_of("anisotropic_max").map(&str_to_anisotropic_max);
+
+        // Match &str to FilterMethod
+        let str_to_filtermethod = |s: &str| {
+            match s {
+                "scale" => FilterMethod::Scale,
+                "mipmap" => FilterMethod::Mipmap,
+                "bilinear" => FilterMethod::Bilinear,
+                "trilinear" => FilterMethod::Trilinear,
+                "anisotropic" => FilterMethod::Anisotropic(anisotropic_max.unwrap()),
+                _ => FilterMethod::Bilinear,
```

Comment:
What do you think would make a better default, Scale or Bilinear? I'm leaning more towards scale (i.e. nearest neighbor) but I could be convinced otherwise.

Reply:
Well I just checked and on shadertoy.com it appears as though mipmapping is now enabled by default, so maybe mipmap?

Reply:
defaults on shadertoy.com when I add a texture

![image](https://user-images.githubusercontent.com/389782/155031056-2c601c85-2e5d-4907-a160-b8528cebadc2.png)

Reply:
(this has me rethinking the wrap mode default too :sweat_smile:)

Reply:
Ok let's do mipmap then, keeping similar defaults as shadertoy seems like the right move here.